### PR TITLE
[Mission] CoP 2-5 Ancient Vow Fixes

### PR DESCRIPTION
**Pull request by Frankie-hz**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

- Added immunity to sleep/lullaby on Mammet-19 Epsilon
- Fixed Starting positions for all BCNM instances
- Mammets in staff form will now cast spells
- Adjusted magic cooldown

## Steps to test these changes

1) !addmission 6 248
2) !zone Monarch Linn
3) Enter BCNM
4) Fight the mammets

### CODE REVIEW

**Comment by TeoTwawki** [comment]
in retail, which message do you get when a sleep attempt is made on these? (I'm assuming you tried this yourself or have a solid source for which method these things use to avoid sleep, as there are multiple)

**Comment by The-Aerec** [comment]
> in retail, which message do you get when a sleep attempt is made on these? (I'm assuming you tried this yourself or have a solid source for which method these things use to avoid sleep, as there are multiple)

Definitely either: Immune or HIGHLY HIGHLY resistant. I have someone running through CoP on retail now. So we should have our answer to the exacts soon.

**Comment by TeoTwawki** [comment]
> I have someone running through CoP on retail now. So we should have our answer to the exacts soon.

excellent :+1: 

I'd bet it probably is immunity, but we always want to be sure. And the log message will tell us.

**Comment by The-Aerec** [comment]
> 

Yea in all the videos online, 0 people even take a sleeper. Its all evasion tanking and pet tanking. (Im guessing bind immune too, but we will see)

**Comment by TeoTwawki** [comment]
> Yea in all the videos online, 0 people even take a sleeper. Its all evasion tanking and pet tanking. (Im guessing bind immune too, but we will see)

Allegedly bind lands but we'll see.

**Comment by claywar** [comment]
> > Yea in all the videos online, 0 people even take a sleeper. Its all evasion tanking and pet tanking. (Im guessing bind immune too, but we will see)
> 
> Allegedly bind lands but we'll see.

Can confirm this.  Used BST as a kiter and that fight, and charm bind landed 100%

**Comment by Frankie-hz** [comment]
> in retail, which message do you get when a sleep attempt is made on these? (I'm assuming you tried this yourself or have a solid source for which method these things use to avoid sleep, as there are multiple)

I have a couple videos of it, need to locate them when I am home later. I did this almost a year ago at this point need to re-dig it up. I have it saved somewhere. Binds can land but its a little resistant, immune to sleep, everything else can land

**Comment by Frankie-hz** [comment]
https://www.youtube.com/watch?v=xT9alFR23Bo&ab_channel=WiggoCaptures

Here is a capture of all of them landing/immune/resist

I'll make sure to post captures/videos/evidence for all the other upcoming PRs I have in the future

**Comment by TeoTwawki** [comment]
> https://www.youtube.com/watch?v=xT9alFR23Bo&ab_channel=WiggoCaptures
> 
> Here is a capture of all of them landing/immune/resist

bless you kind sir

**Review by Xaver-DaRed — APPROVED** [review]

File: sql/mob_spawn_points.sql [hunk 1]
```diff
@@ -6122,15 +6122,15 @@ INSERT INTO `mob_spawn_points` VALUES (16900320,'Shieldtrap','Shieldtrap',20,0,0
...
-INSERT INTO `mob_spawn_points` VALUES (16904193,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-607.789,81.699,-5.607,243);
-INSERT INTO `mob_spawn_points` VALUES (16904194,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-609.317,81.699,1.053,14);
-INSERT INTO `mob_spawn_points` VALUES (16904195,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-598.712,81.699,-1.929,94);
-INSERT INTO `mob_spawn_points` VALUES (16904196,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-4.2,1.534,-6.033,243);
-INSERT INTO `mob_spawn_points` VALUES (16904197,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-5.728,1.534,0.627,14);
-INSERT INTO `mob_spawn_points` VALUES (16904198,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,4.877,1.534,-2.355,94);
-INSERT INTO `mob_spawn_points` VALUES (16904199,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,592.961,-78.254,-5.177,243);
-INSERT INTO `mob_spawn_points` VALUES (16904200,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,591.433,-78.254,1.483,14);
-INSERT INTO `mob_spawn_points` VALUES (16904201,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,602.038,-78.254,-1.499,94);
+INSERT INTO `mob_spawn_points` VALUES (16904193,'Mammet-19_Epsilon','Mammet-19 Epsilon',1,-600.000,82.200,6.176,253);
```
Comment (claywar) [inline sql/mob_spawn_points.sql:6125]: Are these values assumed?  They are very close, but don't appear to match the caps I have from this fight:
16904193 - -600.000,81.699,0.000
16904194 - -600.000,81.900,5.000
16904195 - -600.000,81.699,-5.000

Comment (Frankie-hz) [reply sql/mob_spawn_points.sql:6125]: I did not see coordinates for them in the captures, I must of missed it. I took them from various videos and approximated the locations based on the video evidence i had. I can change them.

**Review by claywar — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**